js/src/reports/programs/campaign-name-cell.scss

```diff
@@ -0,0 +1,5 @@
+.gla-reports__tooltip-campaing-name {
+	.app-tooltip__children-container {
+		width: 100%;
+	}
+}
```

Comment:
💅 / ❓ 

There is a typo in the class name --> `campaing` instead of `campaign`

Besides that, I removed the selector and works the same. What is exactly the intent of the styled wrapper?

Reply:
I fixed the typo: [f2ad4aa](https://github.com/woocommerce/google-listings-and-ads/pull/1496/commits/f2ad4aac518891559f3c66aa0ee3c3abce82dfed)

The intention is to have accessible the tooltip through the full campaign name cell. If I remove the class, the tooltip will show only if we hover over the name.